### ISSUE TITLE
method .query() requires the parameter @values to be a Positional

### ISSUE THREAD

**Issue by frithnanth** [Leont/protocol-postgres#2]
Presently when I  need just one parameter I have to call method `query` this way:
``` raku
$client.query($sql, [42]);
```
because the method is defined as
``` raku
method query(Str $query, @values? --> Promise) 
```
If it were defined as
``` raku
method query(Str $query, *@values --> Promise) 
```
I could use both a scalar or a Positional (or no parameter at all).

**Comment by Leont** [comment]
Yeah is was like that in an earlier version. The problem is that arrays are actually valid values so things got ambiguous.

**Comment by frithnanth** [comment]
Maybe I don't understand what you say, but declaring the value array as `*@values` makes the following examples work:
`$client.query($sql, [42]);`
`$client.query($sql, 42);`
`my @param = 42; $client.query($sql, @param);`
(I cloned your repository, applied my patch, and tested the behavior)
